src/modules/ekf2/EKF2.cpp

```diff
@@ -311,13 +311,12 @@ void EKF2::Run()
 
 	bool imu_updated = false;
 	imuSample imu_sample_new {};
-
+	vehicle_imu_s imu;
 	hrt_abstime imu_dt = 0; // for tracking time slip later
+	imu_updated = _vehicle_imu_sub.update(&imu);
```

Comment:
At the moment we still have to deal with both sensor paths (vehicle_imu and sensor_combined).

I'll try to finish the sensor_combined purge (and multi-EKF replay) in the near future.

Reply:
I need to do a SITL test with multi sensor mode disabled.